app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1>Contestants</h1>
+
+<% @contestants.each do |contestant| %>
+<ul>
+  <li><%= contestant.name %> <br>
+  <%= "Projects: " %>
```

Comment:
No need to put quotes around this.. it will get rendered with quotes if you do this.